docker-compose-aml-web-client.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2019 Banco Santander S.A.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+services:
+  tcf:
+    container_name: tcf
+    image: tcf-dev
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.tcf-dev
+      args:
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - MAKECLEAN=${MAKECLEAN:-1}
+        - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
+        - DISPLAY=${DISPLAY:-}
+        - XAUTHORITY=~/.Xauthority
+    ports:
+      - "1947:1947"
+    volumes:
+      # Below volume mappings are required for DISPLAY settings by heart disease GUI client
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/root/.Xauthority
+    working_dir: "/project/TrustedComputeFramework"
+    entrypoint: "bash -c \"\
+        source scripts/tcs_startup.sh -y \
+        && tail -f /dev/null \""
+    stop_signal: SIGKILL
+    networks:
+      - sgx-network
+
```

Comment:
This docker compose file is similar to what we have docker-compose except network, can we use network setting in existing docker-compse file?

Reply:
The network is only useful for this example. I think it is not worth it to create the network each time TCF is deployed.